vagrant/VERSIONS.sh

```diff
@@ -0,0 +1,12 @@
+# Contains variable definitions for the StudentDashboard Vagrant install.
+# Location of build artifacts on host server.
+ARTIFACTS_SRC=/Users/dlhaines/dev/GITHUB/dlh-umich.edu/FORKS/StudentDashboard
```

Comment:
Is it necessary to use an absolute path that includes your username?  Why not use a relative path based on the "vagrant" directory?